File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/VpsSnapshot.js
goog.provide('API.Client.VpsSnapshot');

/**
 * A VPS snapshot (point-in-time backup) stored on the host node.
 * @record
 */
API.Client.VpsSnapshot = function() {}

/**
 * Snapshot name.
 * @type {!string}
 * @export
 */
API.Client.VpsSnapshot.prototype.name;

/**
 * Disk space used by this snapshot in bytes.
 * @type {!number}
 * @export
 */
API.Client.VpsSnapshot.prototype.used;

/**
 * Unix timestamp of when the snapshot was created.
 * @type {!number}
 * @export
 */
API.Client.VpsSnapshot.prototype.date;

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/VpsSnapshot.scala
package io.swagger.server.model


/**
 * A VPS snapshot (point-in-time backup) stored on the host node.
 *
 * @param name Snapshot name.
 * @param used Disk space used by this snapshot in bytes.
 * @param date Unix timestamp of when the snapshot was created.
 */
case class VpsSnapshot (
  name: Option[String],
  used: Option[Int],
  date: Option[Int]
)

